src/client/Microsoft.Identity.Client/Platforms/netstandard/NetstandardPlatformProxy.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Identity.Client.Core;
+using Microsoft.Identity.Client.Platforms.netcore;
+
+namespace Microsoft.Identity.Client.Platforms.netstandard13
+{
+    /// <summary>
+    /// NetStandard 2.0 is very similar to Net Core. We could completely remove netcore2.1 however
+    /// customers have reported issues with approach.
+    /// </summary>
+    internal class NetstandardPlatformProxy : NetCorePlatformProxy
```

Comment:
This inheritance shows that all the logic in NetCore 2.1 is reused in NetStnandard.